install_config/syncing_groups_with_ldap.adoc

```diff
@@ -886,12 +882,11 @@ The *otheradmins* group has one user member: *Jim*.  This example explains:
 * What the LDAP sync configuration file looks like.
 * What the resulting Group record in {product-title} will be after synchronization.
 
-In the augmented Active Directory schema, both users (Jane and Jim) and groups
-exist in the LDAP server as first-class entries, and group membership is stored
-in attributes on the user or the group. The following snippet of `ldif` defines the
-users and group for this schema:
+In the augmented Active Directory schema, both users (Jane and Jim) and groups exist
+as LDAP entries, and group membership is stored in attributes on the user or the group.
```

Comment:
Change the rest of this in the doc then, I'm pretty sure we use that terminology a lot in the whole thing.